app/server/utils/transfer_limits.py

```diff
@@ -157,160 +151,542 @@ def is_any_token_and_user_is_kyc_verified(credit_transfer):
 def is_any_token_and_user_is_kyc_business_verified(credit_transfer):
     return is_user_and_any_token(credit_transfer) and user_business_or_multidoc_kyc_verified(credit_transfer)
 
-# ~~~~~~LIMIT FILTERS~~~~~~
 
+# ~~~~~~LIMIT FILTERS~~~~~~
 
-@curry
-def after_time_period_filter(days: int, query: Query):
 
+def after_time_period_filter(days: int):
```

Comment:
these are just uncurried version of the original ones. Installed mypy for type hinting and it was WAY more effective at enforcing the function shape